### ISSUE TITLE
build_demo is failing

### ISSUE THREAD

**Issue by AshDevFr** [ProseMirror/prosemirror-tables#42]
Hi, I'd like to try this plugin to see if I can use prosemirror on my project. 
Unfortunately when I try to run the demo I have build issues. 

Do you host this demo somewhere ?

> My config:
> `$ node -v`: v9.3.0
> `$ npm -v`: 5.5.1 

After running a `npm install` then a `npm run build_demo` I have this error:

``` ~/Projects/prosemirror-tables > npm run build_demo    
                                                                                                                                                                                                                                                                                                                                                                                       
> prosemirror-tables@0.6.2 build_demo /Users/user1/Projects/prosemirror-tables
> rollup -c rollup.demo.config.js

(!) Some options have been renamed
https://gist.github.com/Rich-Harris/d472c50732dab03efeb37472b08a3f32
options.entry is now options.input
options.dest is now options.output.file
options.format is now options.output.format

demo.js → demo_bundle.js...
[!] Error: Could not resolve '/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js' from commonjs-proxy:/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js
Error: Could not resolve '/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js' from commonjs-proxy:/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js
    at /Users/user1/Projects/prosemirror-tables/node_modules/rollup-plugin-node-resolve/dist/rollup-plugin-node-resolve.cjs.js:85:23
    at /Users/user1/Projects/prosemirror-tables/node_modules/browser-resolve/index.js:238:20
    at /Users/user1/Projects/prosemirror-tables/node_modules/browser-resolve/index.js:106:24
    at process._tickCallback (internal/process/next_tick.js:152:19)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! prosemirror-tables@0.6.2 build_demo: `rollup -c rollup.demo.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the prosemirror-tables@0.6.2 build_demo script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/user1/.npm/_logs/2018-01-25T18_47_44_670Z-debug.log
```

I tried to reinstall `orderedmap` but it did not change anything.
The file `/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js` exists and is readable

I saw that I needed to change some config in `rollup.demo.config.js`.
Here is the new config I used:

``` 
import buble from "rollup-plugin-buble"
import nodeResolve from "rollup-plugin-node-resolve"
import commonJS from "rollup-plugin-commonjs"

export default {
  input: "demo.js",
  output: {
    file: "demo_bundle.js",
    format: "iife"
  },
  plugins: [
    buble({
      exclude: "node_modules/**",
      namedFunctionExpressions: false
    }),

    nodeResolve({
      main: true,
      browser: true
    }),

    commonJS({
      include: '**',
      sourceMap: false
    })
  ]
}

```

Same common-js error:

```
 ~/Projects/prosemirror-tables > npm run build_demo                                                                                                                                                                                                                                                                                                                                                                                        

> prosemirror-tables@0.6.2 build_demo /Users/user1/Projects/prosemirror-tables
> rollup -c rollup.demo.config.js


demo.js → demo_bundle.js...
[!] Error: Could not resolve '/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js' from commonjs-proxy:/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js
Error: Could not resolve '/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js' from commonjs-proxy:/Users/user1/Projects/prosemirror-tables/node_modules/orderedmap/index.js
    at /Users/user1/Projects/prosemirror-tables/node_modules/rollup-plugin-node-resolve/dist/rollup-plugin-node-resolve.cjs.js:85:23
    at /Users/user1/Projects/prosemirror-tables/node_modules/browser-resolve/index.js:238:20
    at /Users/user1/Projects/prosemirror-tables/node_modules/browser-resolve/index.js:106:24
    at process._tickCallback (internal/process/next_tick.js:152:19)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! prosemirror-tables@0.6.2 build_demo: `rollup -c rollup.demo.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the prosemirror-tables@0.6.2 build_demo script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/user1/.npm/_logs/2018-01-25T18_50_45_277Z-debug.log
```

**Comment by wuzhenyang4165** [comment]
exo? do u have solve this problem? i still build demo failed

**Comment by eshvedai** [comment]
Seems like the problem was in node 9, the issue is fixed in `prosemirror-tables` `0.6.3`
